ReactiveCocoa/Swift/SignalProducer.swift

```diff
@@ -51,12 +51,10 @@ public struct SignalProducer<T, E: ErrorType> {
 	/// then complete, or immediately send an error, depending on the given
 	/// Result.
 	public init(result: Result<T, E>) {
-		switch result {
```

Comment:
New `Result` is not a enum type, so `switch` can't be used. And also the following could't be compiled:

```swift
public init(result: Result<T, E>) {
	result.analysis(ifSuccess: { value in
		self.init(value: value)
	}), ifFailure: { error in
		self.init(error: error)
	}
}
```